c_src/fast_yaml.c

```diff
@@ -151,6 +195,24 @@ static ERL_NIF_TERM zip(ErlNifEnv* env, ERL_NIF_TERM list)
 	return list;
 }
 
+static ERL_NIF_TERM map(ErlNifEnv* env, ERL_NIF_TERM pairs)
+{
+    ERL_NIF_TERM ret;
+    ERL_NIF_TERM key;
+    ERL_NIF_TERM val;
+
+     ret = enif_make_new_map(env);
+    while(enif_get_list_cell(env, pairs, &val, &pairs)) {
+        if(!enif_get_list_cell(env, pairs, &key, &pairs)) {
+            assert(0 == 1 && "Unbalanced object pairs.");
+        }
+        if(!enif_make_map_put(env, ret, key, val, &ret)) {
+            return 0;
```

Comment:
This is only function where there is no space before (, could you fix that?

Reply:
Done